docs/usage.rst

```diff
@@ -56,7 +66,7 @@ Setup a New Bucket
    Next, the user is asked to specify the version of resen-core to use::
 
      Please choose a version of resen-core.
-     Available versions: 2019.1.0, 2020.1.0, 2020.2.0
+     Available versions: 2019.1.0, 2020.1.0, 2020.2.0, 2020.2.0
```

Comment:
2020.2.0 is duplicated. maybe we can add any of the release candidates here 2019.1.0rc1, 2019.1.0rc2. or just eliminate the extra 2020.2.0